packages/cli/src/commands/generate/scaffold/templates/components/Names.tsx.template

```diff
@@ -14,6 +14,26 @@ const DELETE_${singularConstantName}_MUTATION = gql`
 
 const MAX_STRING_LENGTH = 150
 
+const humanize = (value) => {
```

Comment:
Same as above, could we reuse the same `humanize` function as we use in `scaffold.js`?